report_generator/urls_randomizer.py

```diff
@@ -0,0 +1,60 @@
+import sys
+import logging
+import codecs
+import random
+import traceback
+
+
+"""
+urls_randomizer is used to randomize the urls.txt and turls.txt with a given
+seed.
+
+"""
+
+logging.basicConfig(filename='log/error.log',
+                    level=logging.ERROR,
+                    format='%(asctime)s: %(levelname)s: %(message)s')
+
+
+def main():
+    if len(sys.argv) != 3:
+        logging.critical('Wrong number of arguments.',
+                         'Usage: "python urls_randomizer URLS_FILE SEED"')
+        sys.exit(1)
+    urls_file = sys.argv[1]
+    seed = sys.argv[2]
+    try:
+        seed = int(seed)
+    except ValueError:
+        logging.critical('Seed was not an integer')
```

Comment:
I guess you could use sys.exit() here to preserve the original functionality (to not continue if seed was not a proper integer but then again that's not really a stopper).